internal/pkg/describe/pipeline_show.go

```diff
@@ -48,16 +50,17 @@ func NewPipelineDescriber(pipelineName string, showResources bool) (*PipelineDes
 	pipelineSvc := codepipeline.New(sess)
 
 	return &PipelineDescriber{
+		resourceName:  resourceName,
 		pipelineName:  pipelineName,
 		pipelineSvc:   pipelineSvc,
 		showResources: showResources,
-		cfn:           stack.NewStackDescriber(pipelineName, sess),
+		cfn:           stack.NewStackDescriber(resourceName, sess),
```

Comment:
does this work? isn't the stack name different than the pipeline resource name or am I misunderstanding something 😝 ?

Reply:
yes it's different! This is just a result of renaming the variable `pipelineName` to `resourceName` but the value has been the resource name all along.